frontend/src/pages/exploreApplication/EnableModal.tsx

```diff
@@ -114,18 +143,18 @@ const EnableModal: React.FC<EnableModalProps> = ({ selectedApp, shown, onClose }
         </Button>,
       ]}
     >
-      {enable.description ? enable.description : null}
-      {enable.link ? (
+      {enable.description && <div>{enable.description}</div>}
```

Comment:
Very minor nitpick but the following helps reduce clutter
```suggestion
      {enable.description && {enable.description}}
```